pkg/controllers/managementuser/rbac/cluster_handler.go

```diff
@@ -67,47 +67,51 @@ func (h *clusterHandler) sync(key string, obj *v3.Cluster) (runtime.Object, erro
 
 func (h *clusterHandler) doSync(cluster *v3.Cluster) error {
 	_, err := v32.ClusterConditionGlobalAdminsSynced.DoUntilTrue(cluster, func() (runtime.Object, error) {
```

Comment:
Do we need to worry about upgrades? aiui this condition will be set on existing clusters so the below code won't run.

Reply:
We will be okay for an upgrade case since the change to the below code was to force an enqueue of the restricted-admin GRBs. When a user restarts Rancher for the upgrade, the same GRBs will automatically be enqueued